test/e2e_env/multizone/localityawarelb/meshmultizoneservice.go

```diff
@@ -115,7 +116,9 @@ spec:
 	It("should fallback only to first zone", func() {
 		// given traffic to other zones
 		Eventually(responseFromInstance(multizone.KubeZone2), "30s", "1s").
-			MustPassRepeatedly(5).Should(Or(Equal("kube-test-server-1"), Equal("uni-test-server")))
```

Comment:
This doesn't check that both zones are represented, this also passes if it's always `kube-test-server-1`.